public/index.html

```diff
@@ -42,5 +42,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <div id ='modal'></div>
```

Comment:
When using react we want to refrain from changing anything within your HTML page. This should be able to be handled through adding the component into your `App.js` file or through another component file.